### PR TITLE
Release 2.9.2

### PR DESCRIPTION
**Pull request by sb230132**
This release contains bug fixes since the 2.9.1 release.
We recommend that you upgrade at the next available opportunity.

**Bugfixes**
* #5114 Fix issue with deleting data node and dropping the database on multi-node
* #5133 Fix creating a CAgg on a CAgg where the time column is in a different order of the original hypertable
* #5152 Fix adding column with NULL constraint to compressed hypertable
* #5170 Fix CAgg on CAgg variable bucket size validation
* #5180 Fix default data node availability status on multi-node
* #5181 Fix ChunkAppend and ConstraintAwareAppend with TidRangeScan child subplan
* #5193 Fix repartition behavior when attaching data node on multi-node

### CODE REVIEW

File: CHANGELOG.md
Comment (jnidzwetzki) [inline CHANGELOG.md:18]: Could you check the ordering of the entries? It looks like they are not sorted correctly.

File: CHANGELOG.md
Comment (jnidzwetzki) [inline CHANGELOG.md:27]: Please check if we have a `Thanks` for all reported and closed bugs. For example, for #5193 this is missing.

File: CHANGELOG.md
Comment (sb230132) [reply CHANGELOG.md]: Right, now ordered it correctly.

File: CHANGELOG.md
Comment (sb230132) [reply CHANGELOG.md]: #5193 is internal.
Added a thank note for #5181

**Review by jnidzwetzki — APPROVED** [review]
LGTM, but wait for product approval before merging.

File: CHANGELOG.md
Comment (svenklemm) [inline CHANGELOG.md:19]:
```suggestion
* #5133 Fix CAgg on CAgg using different column order than the original hypertable
```

File: CHANGELOG.md
Comment (svenklemm) [inline CHANGELOG.md:23]:
```suggestion
* #5181 Fix ChunkAppend and ConstraintAwareAppend with TidRangeScan child subplan
```

File: CHANGELOG.md
Comment (iroussos) [inline CHANGELOG.md:18]:
```suggestion
* #5114 Fix issue with deleting a data node and dropping the database on multi-node
```

File: CHANGELOG.md
Comment (iroussos) [inline CHANGELOG.md:19]:
```suggestion
* #5133 Fix creating a CAgg on a CAgg where the time column is in a different order of the original hypertable
```

File: CHANGELOG.md
Comment (iroussos) [inline CHANGELOG.md:22]:
```suggestion
* #5180 Fix default data node availability status on multi-node
```

File: CHANGELOG.md
Comment (iroussos) [inline CHANGELOG.md:24]:
```suggestion
* #5193 Fix repartition behavior when attaching data node on multi-node
```

**Review by iroussos — APPROVED** [review]
Thank you @sb230132, I have added some small suggestions, but other than those the PR looks good to me!

**Pull request merged**
